### DISCUSSION TITLE
hello can anyone help out? My page got no response.

### DISCUSSION THREAD

**Discussion by yihenc** [open-webui/open-webui#3401] [General]
I am new. Maybe it's my fault but I need some guidance.
The page got no response and I tried the way to connect Ollama but It didn't work.
**By the way, in the setting, it can verify connection and update models.** I don't know what's going on.
Could anyone kind help me? 
![1](https://github.com/open-webui/open-webui/assets/48312380/4563f9e2-ffe9-4b0a-9915-206b6e64af88)

![image](https://github.com/open-webui/open-webui/assets/48312380/bb87ad2f-b577-423c-9ce8-e1d243e30cab)By 

![2](https://github.com/open-webui/open-webui/assets/48312380/d9c2c62d-a612-45e0-82ac-77b9f0ccc7c6)
![3](https://github.com/open-webui/open-webui/assets/48312380/45af41fa-5f3b-42cc-ad33-d6a4fe29d471)

**Comment by flefevre** [comment]
how do you start open webui? and ollama servers?
if with docker, could you provide the log trace of both servers?

**Reply by yihenc** [reply]
Problem solved by integrate both into one docker. Thanks a lot !
